src/launcher/features/sources/AddLegacySourceWarning.tsx

```diff
@@ -36,8 +36,7 @@ export default () => {
                 longer supported.
             </p>
             <p>
-                For the URL you&apos;ve provided{' '}
-                {/* <code>{sourceToAdd}</code> */}, the app will use{' '}
+                For the URL you&apos;ve provided, the app will use{' '}
```

Comment:
left `<code>{sourceToAdd}</code>` part commented so you can explicitly see this change and check if looks good to you @datenreisender